packages.yml

```diff
@@ -4,3 +4,4 @@ packages:
   
   - git: "https://github.com/fivetran/dbt_fivetran_utils.git"
     warn-unpinned: false
+    revision: jr_working
```

Comment:
will update after this is closed https://github.com/fivetran/dbt_fivetran_utils/pull/24